crypto/o_dir.c

```diff
@@ -23,14 +23,14 @@
 #include "internal/o_dir.h"
 
 #define LPDIR_H
-#if defined OPENSSL_SYS_UNIX || defined DJGPP \
-    || (defined __VMS_VER && __VMS_VER >= 70000000)
+#if defined OPENSSL_SYS_UNIX || defined(DJGPP) \
```

Comment:
```suggestion
#if defined(OPENSSL_SYS_UNIX) || defined(DJGPP) \
```

(this will conflict with the DJGPP PR)